src/components/molecules/MapPartygoerOverlay/MapPartygoerOverlay.tsx

```diff
@@ -1,37 +1,34 @@
 import React, { useMemo } from "react";
 
 import { WithId } from "utils/id";
-import { User } from "types/User";
+import { Experience, User } from "types/User";
 
 import UserProfilePicture from "components/molecules/UserProfilePicture";
 
 interface MapPartygoerOverlayProps {
   partygoer: WithId<User>;
-
-  venueId: string;
-  myUserUid: string;
+  position: Experience;
   totalRows: number;
   totalColumns: number;
 
   /** @default false **/
   withMiniAvatars?: boolean;
-
+  myUserUid?: string;
   setSelectedUserProfile: (user: WithId<User>) => void;
 }
 
-export const MapPartygoerOverlay: React.FC<MapPartygoerOverlayProps> = ({
+export const _MapPartygoerOverlay: React.FC<MapPartygoerOverlayProps> = ({
   partygoer,
-  venueId,
+  position,
```

Comment:
What was `venueId` used for in the past?

And how was the old code using `position` for `currentRow`/`currentCol` when seemingly (according to this diff at least) it didn't exist as a prop before?